pdp/storage.go

```diff
@@ -143,6 +143,14 @@ func (c *command) describe() string {
 		}
 	}
 
+	if nil != c.entity {
```

Comment:
Golang doesn't allow assignment in condition so you can do it in natural way `c.entity != nil`.